client/express-checkout/index.js

```diff
@@ -102,11 +102,9 @@ jQuery( ( $ ) => {
 		/**
 		 * Abort the payment and display error messages.
 		 *
-		 * @param {PaymentResponse} payment Payment response instance.
-		 * @param {string} message Error message to display.
+		 * @param {string}  message Error message to display.
 		 */
-		abortPayment: ( payment, message ) => {
-			payment.paymentFailed( { reason: 'fail' } );
+		abortPayment: async ( message ) => {
```

Comment:
Why was the `async` keyword added here? I don’t see any asynchronous operations being performed inside this method.

Reply:
Most likely it was a leftover, thanks for catching it. Fixed in 93ed023.